src/visual/Paper.js

```diff
@@ -42,17 +42,30 @@ export default class Paper extends joint.dia.Paper {
         if (!cellViewT || !cellViewS || cellViewT.id === cellViewS.id) {
           return false;
         }
+
+        const isSuitablePortClass = (magnet) => {
+          const availableClasses = [
+            'port-body',
+            'port-body-empty',
+            'available-magnet',
+          ];
+
+          let result = true;
+          const portClasses = magnet.getAttribute('class').split(/\s+/);
+
+          _.forEach(portClasses, (portClass) => {
+            result = result && (availableClasses.indexOf(portClass) >= 0);
+          });
```

Comment:
Lodash has a great shortcut for such iterations:
```js
result = _.some(portClasses, pc => availableClasses.indexOf(pc) >= 0);
```
Probably, it would be even better to swap arrays (shorter - first). Also, `_.intersection()` is worth mentioning :)